cmd/root.go

```diff
@@ -2,13 +2,16 @@ package cmd
 
 import (
 	_ "embed"
+	"errors"
 	"fmt"
 	"os"
 )
 
 //go:embed embed/help
 var helpTxt string
 
+var UnknownSubommand = errors.New("unknown subcommand")
```

Comment:
there's a typo here...

```suggestion
var UnknownSubcommand = errors.New("unknown subcommand")
```